Bottom: 09de4ee8b2cb1e745dfc3b339695eef4e679766a
Top:    c5c674f5b5fbda69d69ae9879857ac6766ea94f0
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-30 13:18:31 +0100

Refresh of hid-lesstif-hook-up-debug-draw

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index 395cc98..3922fc4 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -3875,6 +3875,7 @@ lesstif_request_debug_draw (void)
 {
   /* Send drawing to the backing pixmap */
   pixmap = main_pixmap;
+  return &lesstif_hid;
 }
 
 static void
